.github/workflows/test.yml

```diff
@@ -136,7 +136,9 @@ jobs:
         uses: technote-space/get-diff-action@v6
         with:
           PATTERNS: |
+            cli/**
```

Comment:
Do we need this? It's already captured by "Go Related changes" and I think that's used when running e2e and integration tests.

https://github.com/vercel/turbo/blob/b2adfc5c52e4b90f5d86333d327d38b7c0ca0c03/.github/workflows/test.yml#L173